cmd/logsEnvironmentComponent.go

```diff
@@ -82,11 +85,16 @@ func logForComponentReplicas(cmd *cobra.Command, apiClient *apiclient.Radixapi,
 			i := 0
 			for componentName, replicas := range componentReplicas {
 				for _, replica := range replicas {
+					now := time.Now()
+					sinceTime := now.Add(-settings.DeltaRefreshApplication)
+					since, _ := strfmt.ParseDateTime(sinceTime.Format(time.RFC3339))
```

Comment:
This reads to me like converting a timestamp to a string and then parsing it again. What is actually happening here?

Reply:
Lazy work from me. Found a better approach now